docs/reference/cluster_manifest.md

```diff
@@ -435,5 +435,12 @@ Those parameters are grouped under the `tls` top-level key.
   client connects with `sslmode=verify-ca` or `sslmode=verify-full`.
   Default is empty.
 
+* **caSecretName**
+  By setting the `caSecretName` value, the ca certificate file defined by the
+  `caFile` will be fetched from this secret instead of `secretName` above.
+  This secret has to hold a file with that name in its root.
```

Comment:
This is not strictly necessary as the user can provide an absolute path to `caFile` and point into the additional volume directly. The `caFile` only gets expanded with the `/tls` prefix if the path is relative.

Reply:
right, it could be done separately using the new additionalVolumeMounts capability and point to it (which is actually used in background).
Still, following the patterns for tls in similar projects, (therefore align), having a ca secret is might be more intuitive.
FYI, your nice ensurePath function has been reused, this time on top of /tlsca path.
btw, if the ca secret is not provided, the rest of functionality remains as is.

Reply:
Right, the extra code complexity might be worth it if it makes things easier for the user. I am not in a position to judge that.